homeassistant/components/arve/sensor.py

```diff
@@ -0,0 +1,141 @@
+"""Support for Arve devices."""
+
+from collections.abc import Callable, Coroutine
+from dataclasses import dataclass
+from datetime import timedelta
+from typing import Any
+
+from asyncarve import Arve
+
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONCENTRATION_PARTS_PER_CUBIC_METER,
+    CONCENTRATION_PARTS_PER_MILLION,
+    PERCENTAGE,
+    UnitOfTemperature,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DATA_ARVE_CLIENT, DOMAIN, LOGGER
+from .entity import ArveDeviceEntity
+
+SCAN_INTERVAL = timedelta(seconds=10)
```

Comment:
that's really often, is it really required to query that often?

Reply:
So the device sends the measurements each 10 seconds, if there is any restriction about scan interval, I could change it

Reply:
Currently you are polling the device every 10 seconds. If the device `sends` the data you should move to a push style approach (but that could also be done later as an improvement). I'm not aware about any limits, but you are putting load on the system of your users by querying so often. The question is, if there is really a benefit in doing that, or, if the data doesn't change that much, wouldn't less frequent polls be enough. Also, since you say it's a `cloud_polling` integration your also putting load on the vendor's system

Reply:
It works so that the device sends the data to our web platform and the integration gets the data from this platform, not directly from the device. In that case should I define the type of integration as `cloud_push` or stick with `cloud_polling` ?